pkg/apiserver/statement/statement.go

```diff
@@ -125,10 +134,19 @@ func (s *Service) overviewsHandler(c *gin.Context) {
 func (s *Service) detailHandler(c *gin.Context) {
 	db := utils.GetTiDBConnection(c)
 	schema := c.Query("schema")
-	beginTime := c.Query("begin_time")
-	endTime := c.Query("end_time")
 	digest := c.Query("digest")
-	detail, err := QueryStatementDetail(db, schema, beginTime, endTime, digest)
+	beginTime, err := strconv.Atoi(c.Query("begin_time"))
+	if err != nil {
+		_ = c.Error(fmt.Errorf("invalid begin_time: %s", err))
+		return
+	}
+	endTime, err := strconv.Atoi(c.Query("end_time"))
+	if err != nil {
+		_ = c.Error(fmt.Errorf("invalid end_time: %s", err))
+		return
+	}
+
+	detail, err := QueryStatementDetail(db, schema, time.Unix(int64(beginTime), 0).Format(layout), time.Unix(int64(endTime), 0).Format(layout), digest)
```

Comment:
no need to format back again,  timestamp can be used all time

Reply:
I tried to pass the beginTime and endTime as time.Time and int64 type, it seems they work neither...do we need to change any  for this condition?

```
Where("summary_begin_time >= ? AND summary_end_time <= ?", beginTime, endTime).
```

Reply:
I think either `UNIX_TIMESTAMP()` function or `FROM_UNIXTIME()` function is needed.